.appveyor.yml

```diff
@@ -21,7 +21,7 @@ install:
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda
   - conda info -a
-  - conda install xtensor=0.16.0 -c conda-forge
```

Comment:
why quantstack?

Reply:
because the build on conda-forge is really long